### PR TITLE
Create Post “ubuntu-team-kodi-ppa-officially-retired/index”

### PR DESCRIPTION
**Pull request by ProfYaffle**
Automatically generated by Decap CMS

### CODE REVIEW

File: src/content/article/ubuntu-team-kodi-ppa-officially-retired/index.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  title: Ubuntu logo
+  alt: Ubuntu Linux logo
+---
+The Team Kodi PPA has long been a staple for Ubuntu (and similar) users wishing to use more recent, and less adulterated, versions of Kodi.
+
+It is with sadness that the team has come to a decision to officially retire the PPA due to the overhead of maintaining it.
+
+We want to express our sincere thanks to **wsnipex** for maintaining it for so many years, tirelessly making it available to countless users in the endeavour of making Kodi more easily accessible for the masses on many of the more popular Linux distributions.
+
+The Linux world is shifting slowly to containers, and so, going forward, the team is looking to utilise Flatpak to hopefully give Linux users a similar, but more maintainable, "pure" Kodi application. We believe this will also allow us to provide greater accessibility from more distributions than the PPA was ever able to.
```
Comment (JohnVeness) [inline src/content/article/ubuntu-team-kodi-ppa-officially-retired/index.md:19]: I think "greater access" is better than "greater accessibility" in this context. Or maybe "provide greater coverage of distributions".

File: src/content/article/ubuntu-team-kodi-ppa-officially-retired/index.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+We have a considerable amount of work to get the Flatpak builds up to the level the PPA provided (stable, nightlies, etc.), so things will develop over time. In the meantime, then, we will be looking to make this more readily available, and update our documentation accordingly to allow users to install and use Flatpak builds more easily. You can, of course, still build from source if you'd prefer (just as people do on many other distros); whatever route you choose, we'd appreciate your feedback to help us improve the process, document the steps, and generally make things as easy as we can for our users.
```
Comment (JohnVeness) [inline src/content/article/ubuntu-team-kodi-ppa-officially-retired/index.md:21]: I'm a bit confused as to what the "this" referred to in "we will be looking to make this more readily available".

File: src/content/article/ubuntu-team-kodi-ppa-officially-retired/index.md (hunk 1 above)
Comment (ProfYaffle) [reply src/content/article/ubuntu-team-kodi-ppa-officially-retired/index.md:19]: Agree

File: src/content/article/ubuntu-team-kodi-ppa-officially-retired/index.md (hunk 2 above)
Comment (ProfYaffle) [reply src/content/article/ubuntu-team-kodi-ppa-officially-retired/index.md:21]: How about:

"In the meantime, then, we will be looking to make this more obvious as an installation method, and update..."?

**Pull request merged**